state/src/state.rs

```diff
@@ -0,0 +1,8 @@
+use gmeta::{metawasm, Metadata};
+use gstd::{prelude::*, ActorId};
+use onchain_nft_io::*;
+
+#[metawasm]
+pub trait Metawasm {
+    type State = <ContractMetadata as Metadata>::State;
+}
```

Comment:
You should copy here the logic from the state query enum from here:
https://github.com/gear-dapps/on-chain-nft/blob/472143a8475c629a120b678d5b4d3bd437c3b6ad/io/src/lib.rs#L26-L40